src/vdom.rs

```diff
@@ -478,6 +491,35 @@ impl VdomInnerExclusive {
                     self.change_list.emit_append_child();
                 }
             }
+            Node::NamespacedElement(
+                namepsace,
+                ElementNode {
+                    tag_name,
+                    listeners,
+                    attributes,
+                    children,
+                },
+            ) => {
+                self.change_list.emit_create_element_ns(tag_name, namepsace);
+                for l in listeners {
+                    unsafe {
+                        registry.add(l);
+                    }
+                    self.change_list.emit_new_event_listener(l);
+                }
+                for attr in attributes {
+                    if attr.name == "xmlns" {
```

Comment:
@fitzgen unfortunately not all attributes underneath a namespaced element (e.g., xmlns under svg) require `setAttributeNS`. It would be a shame to have to raise this concern to the user of dodrio, but currently without this if check if the user uses xmlns as an attribute to svg, they receive a runtime exception.

Reply:
Is it *just* `xmlns` that shouldn't be namespaced, or are there more edge cases here?

Reply:
It's time to dive into the specs :-D

https://www.w3.org/TR/DOM-Level-3-Core/core.html#Namespaces-Considerations 
_In the DOM, all namespace declaration attributes are by definition bound to the namespace URI: "http://www.w3.org/2000/xmlns/". These are the attributes whose namespace prefix or qualified name is "xmlns" as introduced in [XML Namespaces 1.1]._

This sounds like a partial explanation of why this might be failing. I'm going to change the code so that it checks for xmlns as the start of the attribute, since there can also be an optional namespace prefix that qualifies a tag name to that namespace.

I _believe_ when you use a prefix in your namespace, you're then supposed to supply the namespace as setAttributeNS's first argument. However, I couldn't get this to work...